CHANGELOG.md

```diff
@@ -107,29 +110,27 @@ It includes many new rules, including all new techniques introduced in MITRE ATT
 - build: use Python 3.8 for PyInstaller to support consistently running across multiple operating systems including Windows 7 #505 @mr-tz
 - main: correctly match BB-scope matches at file scope #605 @williballenthin
 
-### Changes
+### Breaking Changes
```

Comment:
i think we should have a "breaking changes" section that is near the top, near "new features", and then keep the (non-breaking) "changes" section here. thoughts?

Reply:
I've moved these entries to the other sections. Will move `Breaking Changes` to the top.

Reply:
IMO all changes can be grouped into other sections as "everything changes" - 🧙